classes/controller/class-ensapi-controller.php

```diff
@@ -266,30 +271,33 @@ public function get_supporter_fields() {
 	 */
 	public function get_supporter_questions() {
 		$response['body'] = get_transient( 'ens_supporter_questions_response' );
-		if ( ! $response['body'] ) {
-			$url = self::ENS_SUPPORTER_URL . '/questions';
+		if ( $response['body'] ) {
```

Comment:
We can do the same as here https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/308/files#r449458709